### PR TITLE
fix(cli):Supress printing variables to logs during ingestion failure

### PR DESCRIPTION
**Pull request by atulsaurav**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.

### CODE REVIEW

File: metadata-ingestion/src/datahub/cli/ingest_cli.py [hunk 1]
```diff
@@ -72,6 +72,13 @@ def ingest() -> None:
     default=False,
     help="If enabled, ingestion runs with warnings will yield a non-zero error code",
 )
+@click.option(
+    "--safe",
```
Comment (dexter-mh-lee) [inline metadata-ingestion/src/datahub/cli/ingest_cli.py:76]: Seems like safe doesn't match what it is actually doing. 

Can we rename this to --suppress-error-logs ? to be more explicit about what you are doing here.

File: metadata-ingestion/src/datahub/cli/ingest_cli.py [hunk 2]
```diff
@@ -72,6 +72,13 @@ def ingest() -> None:
...
+    type=bool,
+    is_flag=True,
+    default=False,
+    help="Supress display of credentials in logs by supressing elaborae stacktrace (stackprinter) during ingestion failures",
```
Comment (dexter-mh-lee) [inline metadata-ingestion/src/datahub/cli/ingest_cli.py:80]: Ditto. We should say "Suppress display of error logs during ingestion failures"

File: metadata-ingestion/src/datahub/cli/ingest_cli.py (hunk 1 above)
Comment (atulsaurav) [reply metadata-ingestion/src/datahub/cli/ingest_cli.py]: Thanks for the feedback! Done!

File: metadata-ingestion/src/datahub/cli/ingest_cli.py (hunk 2 above)
Comment (atulsaurav) [reply metadata-ingestion/src/datahub/cli/ingest_cli.py]: Done. Thanks!

**Review by dexter-mh-lee — APPROVED** [review]

**Pull request merged**